SonicMania/Objects/FBZ/FBZSetup.c

```diff
@@ -85,11 +85,13 @@ void FBZSetup_StageLoad(void)
         scanlineY += FBZSetup->positionYMove[i];
     }
 
+#if RETRO_INCLUDE_EDITOR
```

Comment:
If it's here in the original, we keep it that way.
We may have made changes hre and there for RE2 to properly work with the editor, but not only was it changes we assume were made originally as well (though we have no backing,) but accuracy should be preserved where we can, _especially_ if we made changes for compatibility elsewhere.

`RSDK_ACTIVE_VAR` and `RSDK_ENUM_VAR` may be being if-deffed out. If that is the case, the proper fix would be to go and remove that if-def.